src/public/javascripts/pdf.js

```diff
@@ -6083,6 +6083,7 @@ var renderTextLayer = (function renderTextLayerClosure() {
   function appendText(textDivs, viewport, geom, styles) {
     var style = styles[geom.fontName];
     var textDiv = document.createElement('div');
+    textDiv.setAttribute('prevent-default-highlighting-behaviour', true);
     textDivs.push(textDiv);
```

Comment:
you're not really preventing default highlight behaviour right? you're preventing default pdf highlight behaviour. is that correct?

Reply:
yh